format/index.md

```diff
@@ -901,6 +901,21 @@ Multiple related resources can be requested in a comma-separated list:
 GET /articles/1?include=author,comments.author
 ```
 
+Furthermore, related resources can be requested from a relationship endpoint:
+
+```http
+GET /articles/1/relationships/comments?include=comments.author
```

Comment:
There needs to be a way to include the comments too. `GET /articles/1/relationships/comments?include=comments` should return the relationship data as well as the related comments (in `included`).